x/staking/keeper/cons_pubkey.go

```diff
@@ -0,0 +1,75 @@
+package keeper
+
+import (
+	"time"
+
+	"cosmossdk.io/collections"
+	"cosmossdk.io/x/staking/types"
+
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// SetConsPubKeyRotationHistory sets the consensus key rotation of a validator into state
+func (k Keeper) SetConsPubKeyRotationHistory(
+	ctx sdk.Context, valAddr sdk.ValAddress,
+	oldPubKey, newPubKey *codectypes.Any, height uint64, fee sdk.Coin,
+) error {
+	history := types.ConsPubKeyRotationHistory{
+		OperatorAddress: valAddr.String(),
```

Comment:
can we avoid using the global here? either pass the string or store it as the raw bytes